mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: str
```

Comment:
Choices should be put in brackets... see 
https://github.com/berkeley-stat222/mousestyles/blob/master/DOCFLOW.md